.github/workflows/black.yml

```diff
@@ -0,0 +1,12 @@
+name: Lint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: psf/black@stable
+        with:
```

Comment:
I actually don't want to black our examples because things like this are useful in the examples:

Ours:
```python
cmap_values = [
    0, 1, 1, 2,
    0, 0, 1, 1,
    2, 2, 3, 3,
    1, 1, 1, 5
]
```

black:
```
cmap_values = [0, 1, 1, 2, 0, 0, 1, 1, 2, 2, 3, 3, 1, 1, 1, 5]
```

So let's set jupyter to false and only check the actual sources, i.e. `./fastplotlib`

Reply:
Ok I removed the jupyter line, it defaults to false, and am trying to direct just to the fastplotlib dir.

once nice thing with fpl this stuff runs really fast!